### DISCUSSION TITLE
Welcome to docs.page Discussions!

### DISCUSSION THREAD

**Discussion by cabljac** [invertase/docs.page#146] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by redfieldchristabel** [comment]
Hi there,

I'm currently utilizing docs.page and have a few inquiries:
- Version Switching Mechanism:
Is there a built-in feature in docs.page that allows users to switch between different documentation versions, similar to how Laravel's documentation operates? Specifically, can docs.page fetch available versions directly from Git tags or branches?

-  Filtering Versions:
If fetching versions from Git is possible, is there a way to filter which tags or branches are displayed to users? For instance, can we apply a regex pattern to include only tags that match a certain format (e.g., v*.*.*)?

-  Persistent Components Across Versions:
Is it feasible to have certain components, like a version selector, remain consistent across all documentation versions? For example, even if a user is viewing an older version, can the version selector still reflect and allow navigation to the latest version?

-  Custom Header Components:
Does docs.page currently support custom header components that can be shared across multiple pages? I understand that per-page custom components are supported via MDX, but is there a way to implement a global header component that persists across all documentation pages?

Appreciate any guidance or insights you can provide on these topics!
